cabal.project

```diff
@@ -143,9 +154,29 @@ source-repository-package
     semantics/small-steps-test
     shelley/chain-and-ledger/dependencies/non-integer
     shelley/chain-and-ledger/executable-spec
+    shelley/chain-and-ledger/shelley-spec-ledger-test
     shelley-ma/impl
+    cardano-ledger-core
+    alonzo/impl
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-node.git
+  tag: b3cabae6b3bf30a0b1b4e78bc4b67282dabad0a6
```

Comment:
we should add a note saying that if this hash changes, some of the other `source-repository-changes` potentially need to be changed as well (to get it in sync with the `cabal.project` of `cardano-node`)